carapace-server/src/main/java/org/carapaceproxy/api/ConfigResource.java

```diff
@@ -47,10 +54,40 @@
 public class ConfigResource {
 
     private static final Logger LOG = Logger.getLogger(ConfigResource.class.getName());
+    private static final String CONFIGURATION_DUMPING_FILENAME = "server.dynamic.config";
 
     @javax.ws.rs.core.Context
     ServletContext context;
 
+    @GET
+    public Response dump() {
+        StreamingOutput configStream = (output) -> {
```

Comment:
the result is not so big, we can simply use a String and not the StreamingOutput.
I would be fine to use StreamingOutput but you have not to create the String.

Having the String allows the server to send the Content-Length header and this way it is clear to understand if the file has been downloaded completely